test/powershell/Modules/Microsoft.PowerShell.Utility/Get-FileHash.Tests.ps1

```diff
@@ -77,4 +77,19 @@ Describe "Get-FileHash" -Tags "CI" {
             $result.Path | Should -Be $testDocument
         }
     }
+
+    Context "File should be closed before Get-FileHash writes pipeline output" {
+        It "Should be able to edit the file without 'file is in use' exceptions" {
+            # This test runs against a copy of the document
+            # because it involves renaming it,
+            # and that might break tests added later on.
+            $testDocumentCopy = "${testDocument}-copy"
+            Copy-Item -Path $testdocument -Destination $testDocumentCopy
+
+            $newPath = Get-FileHash -Path $testDocumentCopy | Rename-Item -NewName {$_.Hash} -PassThru -ErrorAction SilentlyContinue
+            Test-Path -Path $newPath.FullName -ErrorAction SilentlyContinue | Should -Be $true
+
+            Remove-Item -Path $testDocumentCopy -ErrorAction SilentlyContinue
```

Comment:
We shouldn't use `SilentlyContinue` without explicit needs. This can hide an error useful for troublethooting. Also we can use `-BeTrue`.
```suggestion
            $newPath = Get-FileHash -Path $testDocumentCopy | Rename-Item -NewName {$_.Hash} -PassThru
            Test-Path -Path $newPath.FullName | Should -BeTrue

            Remove-Item -Path $testDocumentCopy -Force
```

Reply:
We can skip `Test-Path` and just use `$Path | Should -Exist` too.

Reply:
That makes sense about SilentlyContinue - if those throw exceptions, something is going wrong which needs addressing; both suggestions added, thanks :)